Codegen framework output test (#135)
apollographql/apollo-ios-dev: docs/source/development/using-xcframework.mdx

---
title: Using Apollo as an XCFramework
---

> If you need to use Apoll iOS and the code generation output as an XCFramework, this page will give a high level overview of how to do so, as well as collect some tips on things you may encounter that have been reported by the community.

Reviewer: ```suggestion
If you need to use Apollo iOS and the code generation output as an XCFramework, this page will give a high level overview of how to do so, as well as collect some tips on things you may encounter that have been reported by the community.
```

## Creating the XCFramework

### Apollo iOS

> In order to use Apollo iOS as an XCFramework, we provide a [apollo-ios-xcframework]() repository that can generate XCFrameworks from the Apollo iOS code including the necessary commands. This can be used as is or as a starting point for you to customize further for your needs.

Reviewer: Missing the link url to the framework project here.

### Code Generation Output

> You can also use the output from code generation as an XCFramework, you will need to configure this setup yourself, however you can use of [code generation test](https://github.com/apollographql/apollo-ios-dev/tree/main/Tests/TestCodeGenConfigurations/CodegenXCFramework) of this as an example/starting point.

Reviewer: ```suggestion
You can also use the output from code generation as an XCFramework, you will need to configure this setup yourself. You can use the [code generation test](https://github.com/apollographql/apollo-ios-dev/tree/main/Tests/TestCodeGenConfigurations/CodegenXCFramework) project as an example/starting point for this.
```
